src/Conversation/DataClients/Serializers/Normalizers/ImportExport/MessageTemplateNormalizer.php

```diff
@@ -46,6 +51,10 @@ public function denormalize($data, string $type, string $format = null, array $c
             $messageTemplate->setMessageMarkup($data[self::MESSAGE_MARKUP]);
         }
 
+        if (isset($data[self::ORDER])) {
+            $messageTemplate->setMessageMarkup($data[self::ORDER]);
```

Comment:
Think this is a mistake?

```suggestion
            $messageTemplate->setOrder($data[self::ORDER]);
```

Reply:
thanks, yes, that is a mistake!